Library/Formula/openvpn.rb

```diff
@@ -42,11 +45,17 @@ def install
     rm "#{share}/doc/openvpn/README.polarssl"
   end
 
-  def caveats; <<-EOS.undent
-    If you have installed the Tuntap dependency as a source package you will
-    need to follow the instructions found in `brew info tuntap`. If you have
-    installed the binary Tuntap package, no further action is necessary.
+  def caveats
+    if MacOS.version < :yosemite
```

Comment:
Instead of `puts`, assign these to a string and append to it:

```ruby
def caveats
  s = ""

  if MacOS.version < :yosemite
    s += <<-EOS.undent
    If you have installed the Tuntap dependency...
    EOS
  end

  s += <<-EOS.undent
  For OpenVPN to work as a server...
  EOS
end